src/os/ui/layer/vectorlayerui.js

```diff
@@ -146,6 +146,12 @@ os.ui.layer.VectorLayerUICtrl = function($scope, $element, $timeout) {
    */
   this['uniqueId'] = null;
 
+  /**
+   * Feature Toggle
+   * @type {boolean}
+   */
+  this['allowEllipseConfig'] = os.settings.get(os.ui.layer.EllipseColumnsUI.ALLOW_ELLIPSE_CONFIG, false);
```

Comment:
I included this to make the feature toggle-able so that anyone who doesn't want it can disable it in their configs.

Reply:
Ok, we probably want to enable it by default. It does seem like a useful addition, so I would vote making it opt-out instead of opt-in.

Reply:
I was planning to leave this as opt-in because then we don't have to worry about if we missed a config where the change should definitely not be, at least until the feature is done